config.json

```diff
@@ -29,1615 +29,1615 @@
         "version": "22.0.2",
-        "nugetVersion": "122.0.2.7",
+        "nugetVersion": "122.0.2.8",
         "nugetId": "Xamarin.Firebase.ML.Natural.Language.Translate",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-ml-natural-language-translate-model",
         "version": "20.0.9",
-        "nugetVersion": "120.0.9.7",
+        "nugetVersion": "120.0.9.8",
         "nugetId": "Xamarin.Firebase.ML.Natural.Language.Translate.Model",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-ml-vision",
         "version": "24.1.0",
-        "nugetVersion": "124.1.0.7",
+        "nugetVersion": "124.1.0.8",
         "nugetId": "Xamarin.Firebase.ML.Vision",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-ml-vision-automl",
         "version": "18.0.6",
-        "nugetVersion": "118.0.6.7",
+        "nugetVersion": "118.0.6.8",
         "nugetId": "Xamarin.Firebase.ML.Vision.AutoML",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-ml-vision-barcode-model",
         "version": "16.1.2",
-        "nugetVersion": "116.1.2.7",
+        "nugetVersion": "116.1.2.8",
         "nugetId": "Xamarin.Firebase.ML.Vision.BarCode.Model",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-ml-vision-face-model",
         "version": "20.0.2",
-        "nugetVersion": "120.0.2.7",
+        "nugetVersion": "120.0.2.8",
         "nugetId": "Xamarin.Firebase.ML.Vision.Face.Model",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-ml-vision-image-label-model",
         "version": "20.0.2",
-        "nugetVersion": "120.0.2.7",
+        "nugetVersion": "120.0.2.8",
         "nugetId": "Xamarin.Firebase.ML.Vision.Image.Label.Model",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-ml-vision-internal-vkp",
         "version": "17.0.2",
-        "nugetVersion": "117.0.2.7",
+        "nugetVersion": "117.0.2.8",
         "nugetId": "Xamarin.Firebase.ML.Vision.Internal.Vkp",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-ml-vision-object-detection-model",
         "version": "19.0.6",
-        "nugetVersion": "119.0.6.7",
+        "nugetVersion": "119.0.6.8",
         "nugetId": "Xamarin.Firebase.ML.Vision.Object.Detection.Model",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-perf",
-        "version": "20.2.0",
-        "nugetVersion": "120.2.0",
+        "version": "20.3.0",
```

Comment:
This is not just a patch bump, this is a new Java version.

Reply:
I had to manually bump some (remember order of last updates FB was 1st but we merged it as last).
Will fix right now.

Reply:
fixed 120.3.0.1